features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx

```diff
@@ -184,14 +184,15 @@ class Credentials extends React.Component {
      *
      * @memberof Credentials
      */
-    handleSubscribe = () => {
+    handleSubscribe = (updateSubscriptionData) => {
         const promised_subscribe = this.subscribeToApi.createSubscription();
         if (promised_subscribe) {
             promised_subscribe
                 .then((response) => {
                     console.log('Subscription created successfully with ID : ' + response.body.subscriptionId);
-                    this.setState({ open: false });
-                    alert('subscribed successfully');
+                    Alert.info('Subscribed successfully');
+                    if (updateSubscriptionData) updateSubscriptionData();
```

Comment:
AFAIR this style (`if` without curly braces) is not allowed

Reply:
This is how the method is called in other places in the code.That's why i adopted the same approach
Ex: https://github.com/ruwiniwj/carbon-apimgt/blob/d10f765501f55c2031cc8859d55fb930730772ed/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx#L210